api/Element.json

```diff
@@ -2484,7 +2484,8 @@
             },
             "firefox": [
               {
-                "version_added": "24"
+                "version_added": "24",
+                "notes": "Focus lost and this event fired when a non-focusable style is applied to the element after version 110. See <a href='https://bugzil.la/1810077'>bug 1810077</a>."
```

Comment:
Note, the overall behaviour here is referred to as the "focus fixup rule", so we could just put "focus fixup rule supported". However that's not quite as meaningful to the casual reader, and is a superset of the actual change in FF.